functions/src/sgyfetch.ts

```diff
@@ -5,7 +5,7 @@ import {get} from './util/sgyOAuth';
 const SEMESTER = 2; // We are in semester 2
 
 const firestore = admin.firestore();
-const periods = ['1', '2', '3', '4', '5', '6', '7', '8', 'SELF'];
+const periods = ['0','1', '2', '3', '4', '5', '6', '7', '8', 'SELF'];
```

Comment:
Can you include a space between `'0'` and `'1'`? This looks a bit messy without the space.

Reply:
... fine